### DISCUSSION TITLE
Site not secure

### DISCUSSION THREAD

**Discussion by iammrsea** [adonisjs/core#1649] [Help] [answered]
Hello guys, hope everyone is doing great.

Please, could anyone help with a fix with this problem. I have an adonis 5 site deployed on heroku with ssl enabled. 
When I visit the home page [here](https://www.shopjese.com/), the lock icon on the address bar of the browser shows, indicating that's secure but when I visit a page rendered on the server like [this](https://www.shopjese.com/erdman-kutch-and-wisozk), the lock icon doesn't show and it says that the site is insecure even though it's delivered via **https**. Please any ideas what might be the cause of this problem? I'm shitting myself already

**Comment by RomainLanz** [comment, marked as answer]
Hey @iammrsea! 👋 

It only means you have an asset on your page that is not served over `https`, check for any images, css or javascript that it's loaded with `https` and not `http`.

**Reply by iammrsea** [reply]
> Hey @iammrsea! 👋
> 
> It only means you have an asset on your page that is not served over `https`, check for any images, css or javascript that it's loaded with `https` and not `http`.

Thanks man, you are such a life saver.
